### PR TITLE
pkg/littlefs: bump version to v1.6.2

### PR DESCRIPTION
**Pull request by vincent-d**
### Contribution description

Update version to ~~v1.6.1~~v1.6.2 and fix file buffer size.

Upstream version: https://github.com/ARMmbed/littlefs/releases/tag/v1.6.2

### Testing procedure

run tests-littlefs unit tests

### CODE REVIEW

**Comment by x3ro** [comment]
Seems like `1.6.2` is out, with your fix included 🎉

File: pkg/littlefs/Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 PKG_NAME=littlefs
 PKG_URL=https://github.com/geky/littlefs.git
-PKG_VERSION=v1.3
+# v1.6.1
+PKG_VERSION=cb62bf2188854c5b7c44383571ebb19a414e6137
```
Comment (x3ro) [inline pkg/littlefs/Makefile]: Why not use the `v1.6.1` (or `v1.6.2`) tag instead of the full commit hash?

File: pkg/littlefs/Makefile.littlefs [hunk 2]
```diff
@@ -1,6 +1,8 @@
 MODULE := littlefs
 
-CFLAGS += -Wno-unused-parameter -Wno-format
+CFLAGS += -Wno-format
+# TODO: remove when upstream is fixed
```
Comment (x3ro) [inline pkg/littlefs/Makefile.littlefs]: Seems like this is fixed in `v1.6.2` :)

File: pkg/littlefs/Makefile (hunk 1 above)
Comment (vincent-d) [reply pkg/littlefs/Makefile]: I saw some debate on another PR (which I can't remember) about this and I think the decision was to go for the commit hash.

File: pkg/littlefs/Makefile.littlefs (hunk 2 above)
Comment (vincent-d) [reply pkg/littlefs/Makefile.littlefs]: You're more responsive than I am ^^. I will update tomorrow.

File: pkg/littlefs/Makefile (hunk 1 above)
Comment (x3ro) [reply pkg/littlefs/Makefile]: Found [the discussion](https://github.com/RIOT-OS/RIOT/issues/9966). Never mind me, then :)

**Comment by vincent-d** [comment]
Updated with v1.6.2

**Comment by x3ro** [comment]
Looks good and passes on `native` and the `stm32f4-disc1`. On `native`, the tests stall for a noticeable amount of time in the first test (`format`, I presume). This does not happen on the hardware. Do you see an obvious reason for this?

**Comment by vincent-d** [comment]
It took me some time to understand, but I guess you tested on a hardware without any flash. So the MTD emulates a flash in RAM. That's why it's fast to erase and format. On native we emulated a flash with a file, so it take some time to create it, fill it with 0xff and format it.

Everything is normal.

**Comment by jia200x** [comment]
tested in `bl072z-lrwan1` and `native`

**Comment by jia200x** [comment]
@x3ro does it work ok for you? ACK? :)

**Comment by x3ro** [comment]
Absolutely, yes. @vincent-d can you squash?

**Comment by vincent-d** [comment]
@x3ro I kept two commits on purpose, do you prefer I squash them?

**Comment by miri64** [comment]
> @x3ro I kept two commits on purpose, do you prefer I squash them?

I would prefer not to squash them. They address different topics.

**Comment by x3ro** [comment]
Fair enough, for some reason I thought one of them was a fixup yesterday 😅 Apologies. @miri64 Any reason not to merge, then?

**Comment by miri64** [comment]
Depends on how impact the new version is. If it is high, I would rather merge it master after feature freeze for 2018.10, as the high impact feature freeze past on Tuesday.

**Comment by jia200x** [comment]
> Depends on how impact the new version is. If it is high, I would rather merge it master after feature freeze for 2018.10, as the high impact feature freeze passed on Tuesday.

What would you say @vincent-d?

**Comment by vincent-d** [comment]
It's a bugfixes release, so the impact should be low. I guess we can go for the release.

**Comment by jia200x** [comment]
> It's a bugfixes release, so the impact should be low. I guess we can go for the release.

Ok, let's merge it then ;)

**Comment by jia200x** [comment]
thanks!

**Pull request merged**